src/praisonai/praisonai/integrations/managed_agents.py

```
@@ -382,28 +424,56 @@ def _execute_sync(self, prompt: str, stream_live: bool = False) -> str:
-        if stream_live:
-            sys.stdout.write("\n")
-            sys.stdout.flush()
+            with client.beta.sessions.events.stream(session_id) as stream:
+                client.beta.sessions.events.send(
+                    session_id,
+                    events=[{
+                        "type": "user.message",
+                        "content": [{"type": "text", "text": prompt}],
+                    }],
+                )
+                text_parts, tool_log = self._process_events(
```

Ruff (0.15.10)

[warning] Unpacked variable `tool_log` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)